maro/simulator/scenarios/supply_chain/facilities/facility.py

```diff
@@ -203,13 +237,15 @@ def get_node_info(self) -> FacilityInfo:
     def get_sku_cost(self, sku_id: int) -> float:
         # TODO: updating for manufacture, ...
         src_prices: List[float] = [
-            vlt_info.src_facility.skus[sku_id].price
-            for vlt_info in self.upstream_vlt_infos[sku_id]
+            src_facility.skus[sku_id].price
+            for src_facility in self.upstream_facility_list[sku_id]
         ]
         return np.mean(src_prices) if len(src_prices) > 0 else self.skus[sku_id].price
 
     def get_max_vlt(self, sku_id: int) -> int:
         max_vlt: int = 0
-        for vlt_info in self.upstream_vlt_infos[sku_id]:
-            max_vlt = max(max_vlt, vlt_info.vlt)
+        by_fid_and_type = self.upstream_vlt_infos[sku_id]
```

Comment:
(Optional) use `max([info.vlt for by_type in by_fid_and_type.values() for info in by_type.values()])`, a more pythonic way.

Reply:
leave the iteration of upstream_vlt_infos/downstream_vlt_infos like this to be easier to read?

Reply:
OK